src/core/communication.cpp

```diff
@@ -410,7 +410,7 @@ void mpi_gather_stats(int job, void *result, void *result_t, void *result_nb,
     break;
   case 4:
     mpi_call(mpi_gather_stats_slave, -1, 4);
-    predict_momentum_particles((double *)result);
+    predict_momentum_particles((double *)result, local_cells.particles());
```

Comment:
Get from `cell_structure`. In the end the `{local, ghost}_cells` will be removed.